contracts/vaults/BaseVaults/RibbonThetaVault.sol

```diff
@@ -387,8 +432,8 @@ contract RibbonThetaVault is RibbonVault, RibbonThetaVaultStorage {
 
         auctionDetails.oTokenAddress = optionState.currentOption;
         auctionDetails.gnosisEasyAuction = GNOSIS_EASY_AUCTION;
-        auctionDetails.asset = vaultParams.asset;
-        auctionDetails.assetDecimals = vaultParams.decimals;
+        auctionDetails.asset = isUsdcAuction ? USDC : vaultParams.asset;
```

Comment:
Since `isUsdcAuction` reads from storage and is used twice, we can save gas by caching it in a variable first

```solidity
bool _isUsdcAuction = isUsdcAuction;
auctionDetails.asset = _isUsdcAuction ? USDC : vaultParams.asset;
```